sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2548,16 +2548,20 @@ object functions {
   def to_date(e: Column): Column = withExpr { ToDate(e.expr) }
 
   /**
-   * Returns date truncated to the unit specified by the format.
+   * Returns timestamp truncated to the unit specified by the format.
```

Comment:
Ah, thank you for your kind explanation. Will definitely try to avoid such a mistake in the futute.